src/vic3save/Cargo.toml

```diff
@@ -17,6 +17,7 @@ serde = { version = "1", features = ["derive"] }
 thiserror = "1"
 libdeflater = { version = "1.19", optional = true }
 miniz_oxide = { version = "0.7", optional = true }
+clap = { version = "4.0", features = ["derive"] }
```

Comment:
I know this is going to be annoying due to not being able to assign dependencies to only `bin` targets, but can you rewrite `stats_cli` to not use clap, that way downstream CLIs won't need to worry about a conflicting clap version.

Reply:
Oh that's a reason for not using it, annoying.

I'll just remove all the fancy options

Reply:
Nice, can you remove the line in Cargo.toml? No biggie, I may beat you to it if I merge tomorrow.

Reply:
yeah, done